components/script/dom/htmlimageelement.rs

```diff
@@ -298,7 +298,8 @@ impl FetchResponseListener for ImageContext {
 
     fn process_csp_violations(&mut self, _request_id: RequestId, violations: Vec<csp::Violation>) {
         let global = &self.resource_timing_global();
-        global.report_csp_violations(violations);
+        // TODO: Should we set the element here?
```

Comment:
Why would we not?

Reply:
I did some research: I'm guessing it's because the only spec algorithm that sets element is https://www.w3.org/TR/CSP/#should-block-inline, and that's not used by image loads which go through https://www.w3.org/TR/CSP/#fetch-integration and don't have an association with an element. Interesting! Can we write a test that verifies that browsers behave the same and do not fire the event at image elements?

Reply:
Servo does not fetch favicons yet; we only report the icon to the embedder.